### ISSUE TITLE
Incorrect struct naming

### ISSUE THREAD

**Issue by ennerf** [HebiRobotics/MatFileIO#23]
The following code resulted in a struct that had the first field incorrectly named `AB` when loaded in MATLAB 2018b.

```Java
Sink sink = Sinks.newStreamingFile(fileName);
MatFile mat = Mat5.newMatFile()
    .addArray(structName, Mat5.newStruct()
        .set("A", new EjmlSparseWrapper(A))
        .set("B", new EjmlDMatrixWrapper(B)))
    .writeTo(sink);
sink.close();
mat.close();
```

**Comment by ennerf** [comment]
Caused by incorrect implementation of the struct serialization. All names must always include a `NULL TERMINATOR` (`\0`) even if the length is already fully defined from other fields. See [MAT-File Format 1-31](https://www.mathworks.com/help/pdf_doc/matlab/matfile_format.pdf)

**Comment by ennerf** [comment]
Fixed in #24
